php-web/host-crud/mauricio-pereira/src/auth/auth.php

```diff
@@ -0,0 +1,13 @@
+<?php
+    require_once('../database/database.php');
+    $banco = new Database();
+    $senha = $banco->readByNameUser($_POST['login']);
+
+    if (hash('sha256', $_POST['senha']) == $senha['password']) {
```

Comment:
sugiro usar as função password_verify e password_hash  para garantir mais segurança (https://medium.com/@dan.ngandu/how-to-use-password-hash-password-verify-php-part-1-fa78a89985c7)